geo_activity_playground/core/activity_parsers.py

```diff
@@ -120,13 +123,26 @@ def read_fit_activity(path: pathlib.Path, open) -> pd.DataFrame:
                             row["altitude"] = fields["altitude"]
                         if "enhanced_altitude" in fields:
                             row["altitude"] = fields["enhanced_altitude"]
+                        if "grade" in fields:
+                            row["grade"] = fields["grade"]
                         if "speed" in fields:
                             row["speed"] = fields["speed"]
                         if "enhanced_speed" in fields:
                             row["speed"] = fields["enhanced_speed"]
+                        if "temperature" in fields:
+                            row["temperature"] = fields["temperature"]
+                        if "gps_accuracy" in fields:
+                            row["gps_accuracy"] = fields["gps_accuracy"]
                         rows.append(row)
-
-    return pd.DataFrame(rows)
+                    elif "wkt_name" in fields and "sport" in fields and "sub_sport" in fields:
+                        metadata["wkt_name"] = fields["wkt_name"]
```

Comment:
What is “wkt name”? Can you give examples for the values that it takes?

Reply:
Ah, I've found [some documentation](https://developer.garmin.com/fit/file-types/workout/) for this:

![Screenshot_20240113_102727](https://github.com/martin-ueding/geo-activity-playground/assets/976924/6f96ed45-2b28-47cf-988e-09f5e9536d24)

Reply:
Apparently the contents are just strings. I've also added that to `master` now.